Scala/syntax_test_scala.scala

```diff
@@ -2202,3 +2202,9 @@ foo eq bar
 
 foo ne bar
 //  ^^ keyword.operator.comparison.scala
+
+{
+   type Apply = F @uncheckedVariance
+   type A
+// ^^^^ storage.type.scala
+//      ^ entity.name.type.scala
```

Comment:
Should there be a `}` on the last line?

Reply:
Very likely yes! It's been a while since I looked at this PR. I'll try to update it.